### PR TITLE
Create de-CH.coffee

### PR DESCRIPTION
**Pull request by Snake210993**
Added more dialect transalations.
Will keep working on it.

### CODE REVIEW

File: app/locale/de-CH.coffee [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-#  new_home:
-#    title: "CodeCombat - Coding games to learn Python and JavaScript"
-#    meta_keywords: "CodeCombat, python, javascript, Coding Games"
-#    meta_description: "Learn typed code through a programming game. Learn Python, JavaScript, and HTML as you solve puzzles and learn to make your own coding games and websites."
-#    meta_og_url: "https://codecombat.com"
-#    become_investor: "to become an investor in CodeCombat"
-#    built_for_teachers_title: "A Coding Game Built with Teachers in Mind"
-#    built_for_teachers_blurb: "Teaching kids to code can often feel overwhelming. CodeCombat helps all educators teach students how to code in either JavaScript or Python, two of the most popular programming languages. With a comprehensive curriculum that includes six computer science units and reinforces learning through project-based game development and web development units, kids will progress on a journey from basic syntax to recursion!"
-#    built_for_teachers_subtitle1: "Computer Science"
+ # new_home:
```
Comment (mrfinch) [inline app/locale/de-CH.coffee:3]: uncomment (remove `#`). You need to uncomment parent key which is new_home here for child keys(title, meta_keywords etc) to be active

File: app/locale/de-CH.coffee [hunk 2]
```diff
@@ -198,30 +198,30 @@ module.exports = nativeDescription: "Dütsch (Schwiiz)", englishDescription: "Ge
 #    million: "__num__ Million"
 #    billion: "__num__ Billion"
 
-  nav:
-#    educators: "Educators"
-#    follow_us: "Follow Us"
-#    general: "General"
-#    map: "Map"
+# nav:
```
Comment (mrfinch) [inline app/locale/de-CH.coffee:201]: remove #

File: app/locale/de-CH.coffee (hunk 1 above)
Comment (Snake210993) [reply app/locale/de-CH.coffee:3]: Okey, thank you! Will do that.

File: app/locale/de-CH.coffee (hunk 2 above)
Comment (Snake210993) [reply app/locale/de-CH.coffee]: done!

File: app/locale/de-CH.coffee [hunk 3]
```diff
@@ -198,38 +198,38 @@ module.exports = nativeDescription: "Dütsch (Schwiiz)", englishDescription: "Ge
...
+    my_teachers: "Mini Lehrer"
+    careers: "Karriere"
 #    facebook: "Facebook"
 #    twitter: "Twitter"
 #    create_a_class: "Create a Class"
 #    other: "Other"
 #    learn_to_code: "Learn to Code!"
 #    toggle_nav: "Toggle navigation"
-#    schools: "Schools"
+#    schools: "Schuele"
```
Comment (mrfinch) [inline app/locale/de-CH.coffee:232]: remove #

**Review by mrfinch — COMMENTED** [review]
Can you also sign CLA here: https://codecombat.com/cla for me to merge yr code?

**Comment by Snake210993** [comment]
> Can you also sign CLA here: https://codecombat.com/cla for me to merge yr code?

I keep getting an Error.. I write my Github name press save and it fails.

image

**Comment by mrfinch** [comment]
@Snake210993 , thanks for the bug report. I have pushed the fix for it, can u try to submit?

**Comment by Snake210993** [comment]
@mrfinch it worked now :) thanks!

**Review by mrfinch — APPROVED** [review]

**Comment by mrfinch** [comment]
@Snake210993 , there were 2 bugs which I found out after merging this which I fixed: https://github.com/codecombat/codecombat/pull/6532 , FYI.

**Comment by Snake210993** [comment]
> @Snake210993 , there were 2 bugs which I found out after merging this which I fixed: #6532 , FYI.

Thank you very much!

**Pull request merged**